src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -322,6 +330,7 @@ public function getPreferredModulesAction()
      */
     public function importModuleAction(Request $request)
     {
+        $translator = $this->get('translator');
```

Comment:
I agree with you to have only one way to get the translator.

However, I would prefer to have a variable which stores the loaded object. You never know what happen behind, so reusing the same variable in the same function will be slightly faster, and is in my opinion easier to read. :)